as/meta.thrift

```diff
@@ -3,7 +3,14 @@ struct HealthStatus {
     2: optional string message
 }
 
+struct ThriftIDLs {
+    // map: filename -> contents
+    1: required map<string, string> idls
+    // the entry IDL that imports others
+    2: required string entryPoint
```

Comment:
Good point. Need to land this first:
https://github.com/uber/tchannel/pull/1390